dev/diffs/4.0.0.diff

```diff
@@ -3021,7 +3338,6 @@ index ed2e309fa07..a1fb4abe681 100644
 +      conf
 +        .set("spark.sql.extensions", "org.apache.comet.CometSparkSessionExtensions")
 +        .set("spark.comet.enabled", "true")
-+        .set("spark.comet.parquet.respectFilterPushdown", "true")
```

Comment:
Should we keep this? `.set("spark.comet.parquet.respectFilterPushdown", "true")`